lib/socket.js

```diff
@@ -84,7 +85,7 @@ class Transport extends EventEmitter {
 
 const socketFactory = (connect) => {
   const resultConnectFn = (...options) => {
-    const callback = common.extractCallback(options);
+    const callback = common.cbExtract(options);
```

Comment:
But I can agree that `connect` is client-side function, client-side code is less durable in termination/restarting meaning.